dask/dataframe/backends.py

```diff
@@ -687,7 +686,11 @@ def tolist_pandas(obj):
     (pd.Series, pd.Index, pd.api.extensions.ExtensionDtype, np.dtype)
 )
 def is_categorical_dtype_pandas(obj):
-    return pd.api.types.is_categorical_dtype(obj)
+    if hasattr(obj, "dtype"):
+        dtype = obj.dtype
+    else:
+        dtype = obj
+    return isinstance(dtype, pd.api.types.CategoricalDtype)
```

Comment:
Nitpick (this is shorter and more consistent with what we're doing elsewhere)

```suggestion
    return isinstance(dtype, pd.CategoricalDtype)
```

Reply:
Yeah this confused me as well (we have a similar ExtensionDtype above) will adjust